README.md

```diff
@@ -1380,6 +1399,39 @@ may be
 Another option maybe be to initially auto-configure the host during installation (ISO
 based, kickstart, etc.), so that the host is connected to a management LAN or VLAN. It
 strongly depends on your environment.
+#### Installation and Usage Guide
```

Comment:
Let's not include basic instructions on how to use Ansible into separate roles. I'd expect users to go through Ansible docs prior to getting to the role.
We ship roles in collection in https://galaxy.ansible.com/ui/repo/published/fedora/linux_system_roles/
Yes, it is possible to install the role by cloning git repo, but the proper way is to get the collection.

Reply:
@spetrosi  sorry i do not  understand so what was expected from the solution(the issue fix)?

Reply:
Is there some other issue that requests this? This is not part of https://github.com/linux-system-roles/network/issues/292

Reply:
And we provide starting info in https://github.com/linux-system-roles/network/blob/main/README-ansible.md

Reply:
issue #238 and i have made also changes on issue  #292 sorry i will make it different PR